packages/core/contracts-ovm/insured-bridge/external/arbitrum/AVM_CrossDomainEnabled.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity ^0.7.6;
+
+abstract contract AVM_CrossDomainEnabled {
+    modifier onlyFromCrossDomainAccount(address l1Counterpart) {
+        require(msg.sender == applyL1ToL2Alias(l1Counterpart), "ONLY_COUNTERPART_GATEWAY");
+        _;
+    }
+
+    uint160 constant offset = uint160(0x1111000000000000000000000000000000001111);
```

Comment:
Can you add a descriptive comment about why arbitrum uses an offset? This section of the docs explains it clearly. https://developer.offchainlabs.com/docs/l1_l2_messages#address-aliasing

Also apparently, there is some `AddressAliasHelper` somewhere in the arbitrum repo that we can import here that effectively implements `applyL1ToL2Alias` but I can't find it.

Reply:
sure. I added a link to the arbitrum docs as well.